spec/bolt/node/winrm_spec.rb

```diff
@@ -186,18 +184,28 @@ def stub_winrm_to_raise(klass, message)
         ).value
       ).to eq(<<SHELLWORDS)
 echo $env:path\r
-\r
 SHELLWORDS
     end
   end
 
+  it "does not deadlock remote scripts that write > 4k to stderr", vagrant: true do
```

Comment:
In local testing, 4k was enough to deadlock the remote side... though I'm not sure if that's fixed within the WinRM PS Host, or if it's configurable / varies by Windows version.

Ideally, I'd grab this value from a config value on the remote host so that it won't accidentally fail in the future, but not sure that I can.

A similar test in the PowerShell module has to write 64k over a local named pipe before deadlocking.